sys/shell/commands/sc_gnrc_6lo_frag_stats.c

```diff
@@ -28,6 +28,8 @@ int _gnrc_6lo_frag_stats(int argc, char **argv)
 #ifdef MODULE_GNRC_SIXLOWPAN_FRAG_VRB
     printf("VRB full: %u\n", stats->vrb_full);
 #endif
```

Comment:
```suggestion
#endif
#if IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_STATS)
```

Reply:
This C-file is only included when `gnrc_sixlowpan_frag_stats` is included so that would be redundant.